ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -741,7 +741,21 @@ def inner(self, y, name=None):
 
     @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
     def mean(self, axis=None, keepdim=False, name=None):
-        return paddle_frontend.mean(self, axis=axis, keepdim=keepdim)
+        return paddle_frontend.mean(self._ivy_array, axis=axis, keepdim=keepdim)
+
+    @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+    def as_complex(self, name=None):
+        if self._ivy_array.shape[-1] != 2:
```

Comment:
```suggestion
        if self.ivy_array.shape[-1] != 2:
```
Let's not access private attributes.